### ISSUE TITLE
[one-cmds] one-build behavior when no virtual env

### ISSUE THREAD

**Issue by cgbahk** [Samsung/ONE#5736]
I built and installed compiler 1e68ce70. It gave me `one-build` binary.

I didn't prepare venv yet, and ran `one-build` and it hang:
(Note `<<< sth` is my comment, not actual log)
```
$ ./one-build

Error: Virtual environment not found. Please run 'one-prepare-venv' command.
<<< hang here... command not returned
```

First I thought it was hang. But it turns out that it was waiting for my 'click'!

```
$ ./one-build

Error: Virtual environment not found. Please run 'one-prepare-venv' command.
<<< I clicked some place!
import-im6.q16: not authorized `argparse' @ error/constitute.c/WriteImage/1037.
<<< I clicked some place!
import-im6.q16: not authorized `configparser' @ error/constitute.c/WriteImage/1037.
<<< I clicked some place!
import-im6.q16: not authorized `os' @ error/constitute.c/WriteImage/1037.
<<< I clicked some place!
import-im6.q16: not authorized `subprocess' @ error/constitute.c/WriteImage/1037.
<<< I clicked some place!
import-im6.q16: not authorized `sys' @ error/constitute.c/WriteImage/1037.
<<< I clicked some place!
import-im6.q16: not authorized `_utils' @ error/constitute.c/WriteImage/1037.
build/install/bin/one-build: line 30: syntax error near unexpected token `('
build/install/bin/one-build: line 30: `def _get_parser():'
<<< Then it finally gave me a command prompt

$ echo $?
2
```

Not sure this is expected behavior

Of course it worked okay when I prepared venv before.

**Comment by llFreetimell** [comment]
This issue is reproduced in my environment too :(

**Comment by seanshpark** [comment]
@mhs4670go , can you take a look at this?

**Comment by llFreetimell** [comment]
I found the reason :(
exit command wasn't in the code

**Comment by llFreetimell** [comment]
This issue is resolved by #5737, close!

**Comment by cgbahk** [comment]
Thanks
```
$ build/install/bin/one-build
Error: Virtual environment not found. Please run 'one-prepare-venv' command.

$ echo $?
255
```
